### ISSUE TITLE
Continuity at end of domain, outside of domain

### ISSUE THREAD

**Issue by Alex-Jordan** [APEXCalculus/APEXCalculusPTX#182]
Even with recent edits to the continuity section in chapter 1, there is still a lingering issue.

There is no definition for what it would mean to be "continuous at c" when c is at the edge of a function's domain. At this point we have what is needed to (for example) say that sqrt(x) is continuous on [0,∞), but asking if sqrt(x) is continuous at 0 leaves me scratching my head if I'm constraining myself to the definitions APEX has offered. And this comes up in exercises 14 and 16.

Also I mind myself wondering about x=2 in exercise 17. It is not in the domain of the function. But the definition for "continuous at c" presupposes that c is in the domain. It reads as if it doesn't apply one way or the other for a c that is not in the domain in the first place. This is somewhat remedied by this [Paragraph](https://opentext.uleth.ca/apex-video/sec_continuity.html#p-851) but the definition itself still sounds like it doesn't have anything to say on the matter.

**Comment by APEXCalculus** [comment]
Could we append a statement to Def. 1.5.7 along the lines of
"If statements 1 & 2 are true, then we say 'f is continuous at x=a.' If
statements 1 & 3 are true, we say 'f is continuous at x=b.'"

It isn't particularly graceful but it addresses Alex's first point.

Not that my experiences settle a matter, but I distinctly recall a prof in
grad school saying "Definitions are 'if-and-only-if' statements." So while
Part 1 of Def. 1.5.1 says "*If* the limit exists, then f is cont. at x=c",
it is to be read as "if and only if."

If Part 1 were stated outside of a definition, then it can be read as
providing sufficient conditions of continuity at a point. Given inside a
definition, it becomes necessary & sufficient. At least, that's how I've
thought of such statements.

This seems like a good place for David to weigh in.

Thanks.

**Comment by sean-fitzpatrick** [comment]
I think it is standard, including in analysis/topology, that continuity at a point assumes that point is in the domain.

So if a function is not defined at a point, it is not continuous there.

It's probably fine, pedagogically speaking, that we assume the domain is an interval, although the domain could in fact be any set of real numbers. (And if the domain is discrete, or contains isolated points, continuity is automatic at those points.)

For other part: I would suggest adding an aside (since APEX doesn't have remarks) along the following lines:

"When the domain of f is a closed interval, of the form [a, b], or [a, \infty), or (-\infty, b], we define continuity at a (or b) in the same way as Definition 1.5.7."

In Definition 1.5.1, we could add a sentence at the end to clarify: "A function is not continuous at any point not in its domain."

**Comment by Alex-Jordan** [comment]
> I think it is standard, including in analysis/topology, that continuity at a point assumes that point is in the domain.
> So if a function is not defined at a point, it is not continuous there.

No argument there. It's just the way Def 1.5.1 is worded doesn't seem consistent with that (even though I believe that is the intent). Your suggestion about Def 1.5.1 is one way to change that.

> "When the domain of f is a closed interval, of the form [a, b], or [a, \infty), or (-\infty, b], we define continuity at a (or b) in the same way as Definition 1.5.7."

I think Def 1.5.7 could be extended to include what you wrote above. Instead of "Continuity on Closed Intervals" it could be titled "Continuity on Closed Intervals and Domain Endpoints" or something.

**Comment by sean-fitzpatrick** [comment]
OK. I'll take a stab at a new definition this afternoon. (It is almost afternoon where I am.)
Unless I'm needed to help build a dock. (The boat kind, not the computer kind.)

**Comment by sean-fitzpatrick** [comment]
Pull request imminent. I've adjusted the wording in 1.5.1 slightly, and added a sentence immediately following the definition to clarify.

For 1.5.7, I couldn't come up with a way to clarify the definition without making it significantly longer, so I modified the paragraph following the definition to say that the one-sided limits in the definition define what it means for something to be continuous from the left/right, but that if the function is not defined outside the closed interval, we can simply say the function is continuous at these points.

I also included two examples to illustrate the difference.

I feel like this is something that @APEXCalculus might have put in an aside, but perhaps this is important enough that it should go in the main text.

In other books I might have made this a remark, but in our case, remark has been repurposed as "key idea".
